sdk/storage/Azure.Storage.Common/tests/Shared/StorageTestBase.cs

```diff
@@ -470,5 +471,20 @@ protected async Task<Stream> CreateLimitedMemoryStream(long size, long maxMemory
             }
             return stream;
         }
+
+#pragma warning disable SA1402 // File may only contain a single type
+    private class TestCredential : TokenCredential
+#pragma warning restore SA1402 // File may only contain a single type
+    {
+            public override ValueTask<AccessToken> GetTokenAsync(TokenRequestContext requestContext, CancellationToken cancellationToken)
+            {
+                return new ValueTask<AccessToken>(GetToken(requestContext, cancellationToken));
+            }
+
+            public override AccessToken GetToken(TokenRequestContext requestContext, CancellationToken cancellationToken)
+            {
+                return new AccessToken("TEST TOKEN " + string.Join(" ", requestContext.Scopes), DateTimeOffset.MaxValue);
```

Comment:
We should delete https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/storage/Azure.Storage.Common/tests/Shared/StorageRecordedTestSanitizer.cs#L61-L111 since we won't be recording any more auth tokens.